.idea/gradle.xml

```diff
@@ -1,22 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think it's better to not delete this file.

Reply:
Done